backend/typescript/graphql/resolvers/entityResolvers.ts

```diff
@@ -11,31 +28,68 @@ const entityResolvers = {
     entities: async () => {
       return entityService.getEntities();
     },
+    file: async (_req: any, { fileUUID }: { fileUUID: string }) => {
+      return fileStorageService.getFile(fileUUID);
+    },
   },
   Mutation: {
     createEntity: async (
       _req: any,
-      { entity }: { entity: EntityRequestDTO },
+      { entity, file }: { entity: EntityRequestDTO; file: Promise<FileUpload> },
     ) => {
-      return entityService.createEntity({
+      let filePath = "";
+      let fileContentType = "";
+      if (file) {
+        const { createReadStream, mimetype, filename } = await file;
+        const uploadDir = "uploads";
+        filePath = `${uploadDir}/${filename}`;
+        fileContentType = mimetype;
+        await writeFile(createReadStream(), filePath);
+      }
+      const newEntity = entityService.createEntity({
```

Comment:
Must await if not returning result directly
```suggestion
      const newEntity = await entityService.createEntity({
```